src/components/create-safe/InfoWidget/index.tsx

```diff
@@ -1,84 +1,92 @@
-import { Box, Button, Card, CardActions, CardContent, CardHeader, SvgIcon, Typography } from '@mui/material'
-import ChevronLeftIcon from '@mui/icons-material/ChevronLeft'
-import { useState } from 'react'
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Box,
+  Card,
+  CardContent,
+  CardHeader,
+  IconButton,
+  SvgIcon,
+  Typography,
+} from '@mui/material'
 import type { AlertColor } from '@mui/material'
 import type { ReactElement } from 'react'
-
 import LightbulbIcon from '@/public/images/common/lightbulb.svg'
-
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
 import css from './styles.module.css'
 
-type Props = {
+type InfoWidgetProps = {
   title: string
   steps: { title: string; text: string }[]
   variant: AlertColor
+  startExpanded?: boolean
 }
 
-const InfoWidget = ({ title, steps, variant }: Props): ReactElement | null => {
-  const [activeStep, setActiveStep] = useState(0)
-  const [dismissed, setDismissed] = useState(false)
-
-  const isFirst = activeStep === 0
-  const isLast = activeStep === steps.length - 1
-
+const InfoWidget = ({ title, steps, variant, startExpanded = false }: InfoWidgetProps): ReactElement | null => {
   const isMultiStep = steps.length > 1
 
-  const onPrev = () => {
-    if (!isFirst) {
-      setActiveStep((prev) => prev - 1)
-    }
-  }
-
-  const onNext = () => {
-    if (isLast) {
-      setDismissed(true)
-    } else {
-      setActiveStep((prev) => prev + 1)
-    }
-  }
-
-  if (dismissed) {
+  if (steps.length === 0) {
     return null
   }
 
   return (
     <Card sx={{ backgroundColor: ({ palette }) => palette[variant]?.background }}>
       <CardHeader
-        className={css.header}
+        sx={{ paddingBottom: '0px' }}
         title={
-          <Box className={css.headerWrapper}>
-            <Typography
-              variant="caption"
-              className={css.title}
-              sx={{
-                backgroundColor: ({ palette }) => palette[variant]?.main,
-              }}
-            >
-              <SvgIcon component={LightbulbIcon} inheritViewBox fontSize="inherit" className={css.lightbulb} />
-              {title}
-            </Typography>
-            {isMultiStep && (
+          <>
+            <Box className={css.headerWrapper}>
+              <Typography
+                variant="caption"
+                className={css.title}
+                sx={{
+                  backgroundColor: ({ palette }) => palette[variant]?.main,
+                }}
+              >
+                <SvgIcon component={LightbulbIcon} inheritViewBox fontSize="inherit" className={css.lightbulb} />
+                {title}
+              </Typography>
               <Typography variant="caption" className={css.count}>
-                {activeStep + 1} of {steps.length}
+                {steps.length} tip{isMultiStep && 's'}
               </Typography>
```

Comment:
I would remove this. The accordions express the amount of tips there are and "tip" is a bit odd imo. cc @liliiaorlenko

Reply:
make sense! lets remove number of tips everywhere